agent/src/local-e2e/helpers.ts

```diff
@@ -247,11 +247,19 @@ export class LocalSGInstance {
                 .fetchSourcegraphAPI<APIResponse<JSONCLLMSiteConfig>>(q)
                 .then(resp => {
                     if (isError(resp)) {
-                        resolve({
-                            reason: `Failed to connect to Sourcegraph instance: ${resp.message} `,
-                            fix: 'Something went wrong, if you have a running `sg start dotcom-cody-e2e`, you could try to restart it',
-                        })
-                        return
+                        if (resp.message.includes("401")) {
+                            resolve({
+                                reason: `Sourcegraph instance rejected our access token: ${resp.message}`,
+                                fix: 'In your sourcegraph folder, run "sg db default-site-admin".'
```

Comment:
```suggestion
                                fix: 'In your sourcegraph folder, run `sg db default-site-admin`.'
```
nit: just making it consistent with command fix below